docs/data-modeling.md

```diff
@@ -87,6 +80,8 @@ On a technical level, a model maps to the underlying structures of the data sour
 - In MongoDB, a model maps to a _collection_
 - In REST, a model maps to a _resource_
 
+> **Note**: There is not data source connector for REST APIs yet. However, this will be enabled soon through an open connector interface.
```

Comment:
*no

Reply:
[I've updated the page once more](https://github.com/prisma/prisma2/pull/444/files). I've removed MongoDB and REST APIs from the list but still include a _note_ that models would map to collections and resources in MongoDB and REST APIs. IMO this helps build a mental model for what a model is, wdyt?

Reply:
Yeah, the PR you linked to is much clearer 👍

Reply:
Yeah, the PR you linked to is much clearer 👍